services/supervisord/devcontainer_test.go

```diff
@@ -133,6 +133,9 @@ func TestDevContainer(t *testing.T) {
 				err = ioutil.WriteFile(name, b, 0)
 				assert.NoError(t, err)
 			}
+			// force update supervisor config
```

Comment:
actually, restoring original files isn't enough, it doesnt trigger supervisor config reload and we are in case where original configuration is restored, but not applied. It leads to other tests failing (case with prometheus/victoriametrics mocks).

Reply:
i've removed it, seems like prometheus client mock solved problem with flaky tests.